wikimedia/wikipedia-ios
# Fix variant issues in Notifications Center

**Phabricator:** 
https://phabricator.wikimedia.org/T357591

### Notes
This is a bug I discovered and fixed while working on https://phabricator.wikimedia.org/T357370.

### Test Steps
1. Add a ZH Wiki language variant to app languages in Settings
2. Go to Notifications Center Inbox. Confirm only one ZH language (correct variant) appears. Confirm filtering works correctly.
